openapi/cosmos.yaml

```diff
@@ -332,6 +332,144 @@ paths:
         '504':
           $ref: '#/components/responses/statusGatewayTimeout'
 
+  /mdr/specializations/sdtm/domains:
```

Comment:
What is the purpose of this endpoint. Does the DB already have this data?

Reply:
The purpose is to be able to know what domains can be specified in the query parameter at: /mdr/specializations/sdtm/datasetspecializations
Very similar for categories in the BCs
I guess this list could come from the indexer?